app/scripts/controllers/register.js

```diff
@@ -457,6 +461,7 @@
           vm.markers = angular.copy(vm.markers);
           vm.markers.box.lat = parseFloat(newValue.lat.toFixed(6));
           vm.markers.box.lng = parseFloat(newValue.lng.toFixed(6));
+          vm.markers.box.height = newValue.height;
```

Comment:
@noerw @ubergesundheit should we include also a property to save the floor of a box?
So the user can specify something like "second floor or 2"

Reply:
There was already some discussion going on in https://github.com/sensebox/openSenseMap-API/issues/51

This `height` property as part of the location of a box should be the height above sealevel. 

A field for storing the height above the ground in meters would be a better solution than storing the storey. The conversion of storey to meters should be made in the frontend in my opinion.

In fact, we could add a `placement` object to the box which stores not only the height of the box above ground but also things like orientation (north, east, ...), etc

Reply:
:+1: ok so lets keep this in mind for the future

Reply:
actually it is the height above the age84 ellipsoid, as returned by GPS for example